content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+Ta podstrona to przegląd dokumentacji Reacta oraz powiązanych zasobów.
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** to biblioteka JavaScript służąca do budowy interfejsów. Dowiedz się, czym tak naprawdę jest React, na naszej [stronie głównej](/) lub [w tutorialu](/tutorial/tutorial.html).
```

Comment:
`to biblioteka JavaScript służąca do budowy interfejsów` - jakich interfejsów ? Nie uciekło tu słówko mówiące że chodzi o interfejsy użytkownika/graficzne ?

Reply:
Racja, interfejsy w programowaniu to coś troszkę innego. Trzeba napisać, że chodzi o UI, czyli interfejsy użytkownika.